src/test/ui/error-codes/E0081.stderr

```diff
@@ -29,14 +29,35 @@ LL | enum NegDisEnum {
    | ^^^^^^^^^^^^^^^
 LL |
 LL |     First = -1,
-   |             -- first assignment of `-1`
+   |             -- `-1` assigned here
 LL |
 LL |     Second = -2,
-   |     ----------- assigned discriminant for `Last` was incremented from this discriminant
+   |     ----------- discriminant for `Last` incremented from this startpoint (`Second` + 1 variant later => `Last` = -1)
```

Comment:
Is there a way to have this explanation appear *after* the note on `Last`?

Reply:
We should be able to do something like that, although I'm not that well versed in the diagnostic part of rustc. One thing we could do is split the explanation into its own span:
```
error[E0081]: discriminant value `-1` assigned more than once
 --> tst.rs:5:1
  |
5 | enum NegDisEnum {
  | ^^^^^^^^^^^^^^^
6 |     First = -1,
  |             -- `-1` assigned here
7 |     Second = -2,
8 |     Last,
  |     ---- `-1` assigned here
  |
note: discriminant for `Last` incremented from this startpoint (`Second` + 1 variant later => `Last` = -1)
 --> tst.rs:7:5
  |
7 |     Second = -2,
  |     ^^^^^^^^^^^

error: aborting due to previous error
```
The drawback is that this adds slightly to diagnostic clutter.